From d9b29b093f4ed04fb5c1fe505d81549d08ca9893 Mon Sep 17 00:00:00 2001
From: Anthony Iliopoulos <ailiop@altatus.com>
Date: Mon, 3 Feb 2020 14:48:40 +0100
Subject: [PATCH] liburing: add missing header include for mode_t

The typedef mode_t is defined in sys/types.h which is not being
explicitly included in liburing.h, and this breaks musl compilation.
---
 src/include/liburing.h | 1 +
 1 file changed, 1 insertion(+)

diff --git src/include/liburing.h src/include/liburing.h
index 78e8364..992fe7c 100644
--- src/include/liburing.h
+++ src/include/liburing.h
@@ -10,6 +10,7 @@ extern "C" {
 #include <signal.h>
 #include <inttypes.h>
 #include <time.h>
+#include <sys/types.h>
 #include "liburing/compat.h"
 #include "liburing/io_uring.h"
 #include "liburing/barrier.h"
-- 
2.25.0

